ASSEMBLY_REPORT.md

```diff
@@ -686,4 +662,28 @@ concepts/what-is-the-temporal-platform -> /temporal#temporal-platform
 
 app-dev-context/metrics -> /application-development/observability#metrics
 
+cli/cmd-options/address -> /cli/cmd-options#address
```

Comment:
can we sort this list so that you can tell what actually changed vs just changed location in the file?

Reply:
I didn't build the assembly report, but I can ask and see if this would be worth implementing. 

I could see the report ballooning on major edits, so I'm a bit iffy about your suggestion going in. I get it as an alternative to scrolling through GitHub, but it would also be redundant since there's already something to show what actually changed.

Reply:
merged from main. Cully made some updates to collapse this

Reply:
makes sense, thanks!